packages/odyssey-react/src/components/Select/Select.module.scss

```diff
@@ -208,15 +208,18 @@
     @include border-radius(100%);
 
     display: inline-block;
-    position: relative;
+    position: absolute;
+    inset-block-start: 50%;
```

Comment:
always takes me longer to understand CSS written with logical operators still need some more exposure feor this to feel natural 😅